tests/CMakeLists.txt

```diff
@@ -6,13 +6,14 @@ ADD_DEFINITIONS(-DBOOST_TEST_DYN_LINK)
 # Tests to run. Tests can be run using x, y, and z processors using
 # MFMG_ADD_TEST(test x y z)
 MFMG_ADD_TEST(test_laplace 1 2 4)
-MFMG_ADD_TEST(test_agglomerate 1 2 4)
+MFMG_ADD_TEST(test_hierarchy 1 2 4)
+# MFMG_ADD_TEST(test_agglomerate 1 2 4)
 MFMG_ADD_TEST(test_eigenvectors 1)
-MFMG_ADD_TEST(test_restriction_matrix 1 2 4)
+# MFMG_ADD_TEST(test_restriction_matrix 1 2 4)
 
-IF(${MFMG_ENABLE_CUDA})
-  MFMG_ADD_CUDA_TEST(test_utils 1 2 4)
-  MFMG_ADD_CUDA_TEST(test_eigenvectors_device 1)
-  MFMG_ADD_CUDA_TEST(test_restriction_matrix_device 1)
-  MFMG_ADD_CUDA_TEST(test_sparse_matrix_device 1 2 4)
-ENDIF()
+# IF(${MFMG_ENABLE_CUDA})
```

Comment:
Why did you have to comment this? It should not have been needed.

Reply:
I'm disabling this just in case. I'll re-enable in the final patch.